app/src/main/java/com/wittyapp/MyApplication.kt

```diff
@@ -0,0 +1,19 @@
+package com.wittyapp
+
+import android.app.Application
+import com.facebook.FacebookSdk
+import com.facebook.appevents.AppEventsLogger
+
+/**
+ * Created by bett on 8/22/17.
```

Comment:
Who is bett? Did you secretly hire a volunteer 2 years in the past?

Reply:
You caught me. 
The seed to every thought you ever had related to volunteering was planted by me. I exactly knew what we will decide now, 2 years back. I had some spare time then and so I decide I will outsource the dev work to Bett. I chopped his hands once he finished his work. Have a nice day.